offload-build.in

```diff
@@ -92,6 +93,12 @@ makepkg --source || die "unable to make source package"
 # but an empty src dir is created in PWD. Remove once fixed in makepkg.
 rmdir --ignore-fail-on-non-empty src 2>/dev/null || true
 
+# Find the most specific valid makepkg.conf file
+makepkg_config="@pkgdatadir@/makepkg-${arch}.conf"
+if [[ -f @pkgdatadir@/makepkg-${repo}-${arch}.conf ]]; then
```

Comment:
This is looking for the makepkg-repo-arch.conf file on the local machine, where it has no reason to be installed, instead of the build server?